internal/test/wmcb/powershell/wget-ignore-cert.ps1

```diff
@@ -24,4 +24,5 @@ public static class Dummy {
 }
 [System.Net.ServicePointManager]::ServerCertificateValidationCallback = [dummy]::GetDelegate()
 
+$ProgressPreference = "SilentlyContinue"
```

Comment:
Why are you making this change?

Reply:
After switching to SSH, the powershell script stopped working. as mentioned in the commit message made this change to make sure the script runs successfully. Was getting the following error before this change:
```
wget : Win32 internal error "Access is denied" 0x5 occurred while reading the console output buffer. Contact Microsoft 
Customer Support Services.
At C:\Temp\wget-ignore-cert.ps1:27 char:1
+ wget -UserAgent $useragent $server -o $output
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ReadError: (:) [Invoke-WebRequest], HostException
    + FullyQualifiedErrorId : ReadConsoleOutput,Microsoft.PowerShell.Commands.InvokeWebRequestCommand
```

Reply:
If this change is actually needed, it cannot go in as part of this PR. Anyhow it looks like #232 will make this irrelevant.

Reply:
As #232 is closed, should i create another PR for this change ?

Reply:
Yes

Reply:
Does this block your PR from merging? If yes, please leave it as it is. Ideally this should have been part of a commit that migrated from WinRM to SSH.

Reply:
yes, 
this should go in first in order for this PR to merge. 
this was originally part of the commit that migrated from WinRM to SSH. But, as that commit was after this commit. i had to squash it to reduce confusion while reviewing.

Reply:
Yes, you fist migrated to MachineSets and then migrated to WinRM to SSH. You should have opened a separate PR to first move from WinRM to SSH or at least made it your first commit in this PR.